kie-server-parent/kie-server-services/kie-server-services-kafka/src/main/java/org/kie/server/services/jbpm/kafka/KafkaServerExtension.java

```diff
@@ -83,7 +84,13 @@ public void init(KieServerImpl kieServer, KieServerRegistry registry) {
             logger.warn("Kafka extension already initialized");
             return;
         }
-
+        
+        // register work item handler
+        String desc = System.getProperty(DESCRIPTOR);
+        if(desc == null ) {
+            System.setProperty(DESCRIPTOR, "classpath:/META-INF/kafka-deployment-descriptor-defaults.xml");
```

Comment:
you are overriding this property therfore prometheus won't go in.

Reply:
thats the property that is used by kie server here https://github.com/kiegroup/droolsjbpm-knowledge/blob/master/kie-internal/src/main/java/org/kie/internal/runtime/manager/deploy/DeploymentDescriptorManager.java#L61
I think is better to use initial proposal and avoid this clash

Reply:
Initial proposal depends on JBPM behaviour while template is kie server one, so extending DeploymentDescriptorManager capabilities (adding add static method) to allow several templated